pom.xml

```diff
@@ -671,6 +671,527 @@
+                <artifactId>saxon</artifactId>
+                <version>${saxon.orbit.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>net.sf.saxon</groupId>
+                        <artifactId>saxon-xpath</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>org.htmlcleaner.wso2</groupId>
+                <artifactId>htmlcleaner</artifactId>
+                <version>${htmlcleaner.orbit.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>htmlcleaner</groupId>
+                        <artifactId>htmlcleaner</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>commons-dbcp.wso2</groupId>
+                <artifactId>commons-dbcp</artifactId>
+                <version>${commons-dbcp.orbit.version}</version>
+            </dependency>
+<!--            <dependency>-->
+<!--                <groupId>commons-pool.wso2</groupId>-->
+<!--                <artifactId>commons-pool</artifactId>-->
+<!--                <version>${commons-pool.orbit.version}</version>-->
+<!--            </dependency>-->
+            <dependency>
+                <groupId>io.dropwizard.metrics</groupId>
+                <artifactId>metrics-core</artifactId>
+                <version>${metrics.core.version}</version>
+            </dependency>
+<!--            <dependency>-->
+<!--                <groupId>io.netty</groupId>-->
+<!--                <artifactId>netty-handler</artifactId>-->
+<!--                <version>${netty.version}</version>-->
+<!--            </dependency>-->
+<!--            <dependency>-->
+<!--                <groupId>io.netty</groupId>-->
+<!--                <artifactId>netty-buffer</artifactId>-->
+<!--                <version>${netty.version}</version>-->
+<!--            </dependency>-->
+<!--            <dependency>-->
+<!--                <groupId>io.netty</groupId>-->
+<!--                <artifactId>netty-common</artifactId>-->
+<!--                <version>${netty.version}</version>-->
+<!--            </dependency>-->
+<!--            <dependency>-->
+<!--                <groupId>io.netty</groupId>-->
+<!--                <artifactId>netty-transport</artifactId>-->
+<!--                <version>${netty.version}</version>-->
+<!--            </dependency>-->
+<!--            <dependency>-->
+<!--                <groupId>io.netty</groupId>-->
+<!--                <artifactId>netty-codec</artifactId>-->
+<!--                <version>${netty.version}</version>-->
+<!--            </dependency>-->
+            <dependency>
+                <groupId>org.wso2.orbit.org.antlr</groupId>
+                <artifactId>antlr4-runtime</artifactId>
+                <version>4.5.1.wso2v1</version>
```

Comment:
let's define a property for this or use the existing one if there are any

Reply:
Done